docs/concepts/storage/storage-classes.md

```diff
@@ -52,6 +52,20 @@ mountOptions:
   - debug
 ```
 
+### Volume Binding Mode
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+
+This field requires the `VolumeScheduling` feature gate to be enabled.
```

Comment:
Link for how to enable?

Reply:
I don't think we have a page describing how to enable feature gates, but if you search the site, it will quickly point to some generated docs on each of the Kubernetes components and the flags that they take.

Reply:
It might be beneficial to provide a link here so users don't have to search for it themselves.

Reply:
Yeah, the challenge is there isn't just one link that describes the whole sequence.  It's just every single kubernetes component has a help page describing their flags that you can pass.

Reply:
https://github.com/kubernetes/website/pull/6364